main.py

```diff
@@ -138,6 +138,13 @@ def handle_trade(trade: Trade) -> None:
     except ValueError:
         logger.exception('Unable to handle open order')
 
+def get_target_bid(ticker) -> float:
```

Comment:
Please use type hints for `ticker`

Reply:
Should we actually start using stuff from the `typings`? https://docs.python.org/3/library/typing.html

Like here ticker would be `Dict[str, float]` instead of `dict`. Or even a `Ticker` named type derived from `Dict[str, float]` with the `NewType`?

Reply:
Good point, I would prefer `Dict[str, float]` over `dict`, because it is more precise, but I'm not a fan of deriving from it, because it makes the codebase harder to read and to maintain

Reply:
Fair point. I'll use the generic `Dict[str, float]` here now, let's do a separate PR to fix other places.

Reply:
Sounds good, feel free to merge it